### DISCUSSION TITLE
the command node not found

### DISCUSSION THREAD

**Discussion by McGarret** [spatie/laravel-pdf#144] [Q&A]
### What happened?

I'm using spatie/Laravel-pdf because i want to use tailiwnd in my laravel blade project. I see the documention, setting everything up but when i want to download my pdf, there's the error i mentionned .
![Capture d'écran 2024-05-19 102421](https://github.com/spatie/laravel-pdf/assets/53648830/a170b415-fa98-46ec-ac63-450d8826b263)

### How to reproduce the bug

When i click on download pdf button

### Package Version

1.5

### PHP Version

8.3.3

### Laravel Version

10.48.10

### Which operating systems does with happen with?

Windows

**Comment by gullithisouza** [comment]
The same thing is happening to me here, I have the same resources as McGarrett

**Comment by mohamed-salah-tawfik** [comment]
any updates?

**Comment by mohamed-salah-tawfik** [comment]
I found a method named `withBrowsershot` at the `PdfBuilder` that you can use to set the binaries path

````
return pdf()
    ->withBrowsershot(function (Browsershot $shot) {
        $shot->setNodeBinary('YOUR-CUSTOM-PATH/node')
            ->setNpmBinary('YOUR-CUSTOM-PATH/npm');
    })
    ->view('html-view')
    ->name('file-name.pdf')
    ->download();
```
